sympy/solvers/polysys.py

```diff
@@ -429,3 +439,389 @@ def solve_triangulated(polys, *gens, **args):
+          one component of the solution
+        - condition is a Boolean expression that must be satisfied for the system to be solvable
+
+    Examples
+    ========
+
+    >>> from sympy.solvers.polysys import factor_poly_system
+    >>> from sympy.abc import x, y
+
+    A simple system with multiple solutions:
+    >>> systems, cond = factor_poly_system([x**2 - 1, y - 1])
+    >>> systems
+    [[x + 1, y - 1], [x - 1, y - 1]]
+    >>> cond
+    True
+
+    A system with no solution:
+    >>> systems, cond = factor_poly_system([x, 1])
+    >>> systems
+    []
+    >>> cond
+    True
+
+    A system with infinite solutions:
+    >>> systems, cond = factor_poly_system([x - x, (x + 1)**2 - (x**2 + 2*x + 1)])
+    >>> systems
+    [[]]
+    >>> cond
+    True
+
+    Notes
+    =====
+
+    - A return of a list constain an empty list [[]] in the tuple means any value of the symbol(s) is a solution
+    - A return of an empty list [] in the tuple means no solutions for the system exists.
+
+
+    See Also
+    ========
+    factor_system_cond : Returns both factors and conditions in polynomial form
+    factor_system_bool : Returns a Boolean combination representing the solution
+    """
+
+    systems, conds = factor_system_cond(eqs, *gens, **kwargs)
+    systems = [[f.as_expr() for f, c in system] for system in systems]
+    return systems, And(*conds)
+
+
+def factor_system_bool(eqs, *gens, **kwargs):
+    """
+    Takes a system of expressions and returns a Boolean combination of equations
+    that represents the same solution set. The result is in disjunctive normal form
+    (OR of ANDs).
+
+    Parameters
+    ==========
+
+    eqs : list
+       List of expressions to be factored. Each expression is assumed to be equal to zero.
+    *gens : Symbol or sequence of Symbols, optional
+       Generator(s) of the polynomial ring. If not provided, all free symbols will be used.
+    **kwargs : dict, optional
+       Optional keyword arguments
+
+
+    Returns
+    =======
+
+    Expr:
+       A Boolean combination of equations in disjunctive normal form.
+
+    Examples
+    ========
+
+    >>> from sympy.solvers.polysys import factor_system_bool
+    >>> from sympy.abc import x, y, a, b, c
+    >>> factor_system_bool([x**2 - 1])
+    Eq(x - 1, 0) | Eq(x + 1, 0)
+
+    >>> factor_system_bool([x**2 - 1, y - 1])
+    (Eq(x - 1, 0) & Eq(y - 1, 0)) | (Eq(x + 1, 0) & Eq(y - 1, 0))
+
+    >>> eqs = [a * (x - 1), b]
+    >>> factor_system_bool(eqs, [x])
+    Eq(b, 0) & (Eq(a, 0) | Eq(x - 1, 0))
+
+    >>> eqs = [a * x ** 2 - a, b * (x + 1), c]
+    >>> factor_system_bool(eqs, [x])
+    Eq(c, 0) & (Eq(x + 1, 0) | (Eq(a, 0) & Eq(b, 0)))
+
+    >>> factor_system_bool([x**2 + 2*x + 1 - (x + 1)**2])
+    True
+
+    Notes
+    =====
+
+    - Function returns ``True`` when all values of the symbol(s) is a solution.
+    - Function returns ``False`` when the system cannot be solved.
```

Comment:
Move this to an Explanation section that should first explain explicitly that the boolean expression is logically equivalent to the original input equations.